### PR TITLE
Renaming Issue Templates correctly and adding a pull request template.

### PR DESCRIPTION
**Pull request by tariq1890**

### CODE REVIEW

**Comment by tariq1890** [comment]
That's odd. A unit test case is failing. Was it already broken?

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (thomastaylor312) [inline .github/PULL_REQUEST_TEMPLATE.md]: This should link to the Helm contributor guide

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (thomastaylor312) [inline .github/PULL_REQUEST_TEMPLATE.md]: I don't think we really leverage the release note, so we may not want to put it here. Let's see what some of the other maintainers here

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (thomastaylor312) [inline .github/PULL_REQUEST_TEMPLATE.md]: Spelling nit: Should be `i.e.`

**Review by thomastaylor312 — COMMENTED** [review]
Thank you for adding this. I'll want to get several of the other maintainers' opinions on this before merge because this will show up for all PRs

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (tariq1890) [reply .github/PULL_REQUEST_TEMPLATE.md]: Done

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (tariq1890) [reply .github/PULL_REQUEST_TEMPLATE.md]: Ok

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (tariq1890) [reply .github/PULL_REQUEST_TEMPLATE.md]: Done

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (mattfarina) [inline .github/PULL_REQUEST_TEMPLATE.md]: If we start using a release note section we should do two things:

1. Add a label to the issue (automatically?) noting the release notes
1. Make sure the issue is filed in a proper milestone

This way someone performing a release can quickly get a list of notes to include.

Comment (thomastaylor312) [reply .github/PULL_REQUEST_TEMPLATE.md]: Yeah, we would need to get the automation in place for that

Comment (bacongobbler) [reply .github/PULL_REQUEST_TEMPLATE.md]: Yeah, let's remove this for now until someone writes up automation for this.

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (bacongobbler) [inline .github/PULL_REQUEST_TEMPLATE.md]: I'd change these to be a bit more personal:

```suggestion
- [ ] this PR contains documentation
```

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (bacongobbler) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- [ ] this PR contains unit tests
```

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (bacongobbler) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- [ ] this PR has been tested for backwards compatibility
```

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (bacongobbler) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
1. Make sure to read the Contributing Guide before submitting your PR: https://github.com/helm/helm/blob/master/CONTRIBUTING.md
2. If this PR closes another issue, add 'closes #<issue number>' somewhere in the PR summary. GitHub will automatically close that issue when this PR gets merged. Alternatively, adding 'refs #<issue number>' will not close the issue, but help provide the reviewer more context.
```

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (bacongobbler) [inline .github/PULL_REQUEST_TEMPLATE.md]: I'd remove this line in favour of above^

**Review by bacongobbler — CHANGES_REQUESTED** [review]

**Comment by tariq1890** [comment]
I don't remember creating this PR lol. Looking into the comments

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (tariq1890) [reply .github/PULL_REQUEST_TEMPLATE.md]: done

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (tariq1890) [reply .github/PULL_REQUEST_TEMPLATE.md]: done

**Comment by tariq1890** [comment]
Closing this in favor of #5022

**Pull request closed without merging**
